src/contrib/cluster/Akka.Cluster.Tools/Client/ClusterClient.cs

```diff
@@ -390,16 +578,16 @@ private bool Establishing(object message)
 
         switch (message)
         {
+            case DeadLetter { Message: ClusterReceptionist.GetContacts } dl:
+                PruneContacts(dl.Recipient.Path.Address.ToString());
```

Comment:
Dead contact detection, remove contact from contact list if `GetContacts` were redirected to `DeadLetters`